### PR TITLE
Fix typo and update some sentences translation

### PR DESCRIPTION
**Pull request by rootsongjc**
- Refer to #2335 and #2377 which have no update for a long time

### CODE REVIEW

File: content_zh/docs/concepts/traffic-management/index.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ Q: *已经使用容错库（例如 [Hystrix](https://github.com/Netflix/Hystrix)
 
 Q: *同时使用应用级库和 Envoy 时，怎样处理故障？*
 
-假如对同一个目的服务给出两个故障恢复策略（例如，两次超时设置——一个在 Envoy 中设置，另一个在应用程序库中设置），**当故障发生时，两个限制都将被触发**。例如，如果应用程序为服务的 API 调用设置了 5 秒的超时时间，而运维人员配置了 10 秒的超时时间，那么应用程序的超时将会首先启动。同样，如果 Envoy 的熔断器在应用熔断器之前触发，对该服务的 API 调用将从 Envoy 收到 503 错误。
+假如对同一个目的服务给出两个故障恢复策略（例如，两次超时设置——一个在 Envoy 中设置，另一个在应用程序库中设置），**当故障发生时，将触发两者中更严格的那个**。例如，如果应用程序为服务的 API 调用设置了 5 秒的超时时间，而运维人员跟 Envoy 配置了 10 秒的超时时间，那么应用程序的超时将会首先启动。同样，如果 Envoy 的熔断器在应用熔断器之前触发，对该服务的 API 调用将从 Envoy 收到 503 错误。
```
Comment (fleeto) [inline content_zh/docs/concepts/traffic-management/index.md]: 给 Envoy 配置了？

Comment (rootsongjc) [reply content_zh/docs/concepts/traffic-management/index.md]: Fixed

**Comment by rootsongjc** [comment]
@geeknoid This PR only fix some Chinese translations but there is a file not locate at `content_zh`, could you merge this, please.

**Pull request merged**